source/06_monitoring_features/notifications.rst

```diff
@@ -0,0 +1,220 @@
+.. _monitoring_features/notifications:
+
+=============
+Notifications 
+=============
+
+
+Introduction 
+============
+
+.. image:: /_static/images///official/images/objects-contacts.png
+   :scale: 90 %
+
+This chapter will attempt to explain exactly when and how host and service notifications are sent out, as well as who receives them.
+
+Notification escalations are explained :ref:`here <alignak_features/notifications_escalations>`.
+
+
+When Do Notifications Occur? 
+============================
+
+The decision to send out notifications is made in the service check and host check logic. Host and service notifications occur:
+
+    * When a HARD state change occurs.
+    * When a host or service remains in a HARD non-OK state and the time specified by the ``notification_interval`` option in the host or service definition has passed since the last notification was sent out (for that specified host or service).
+
+
+Who Gets Notified? 
+==================
+
+Each host and service definition has a ``contact_groups`` option that specifies which contact groups receive 
+notifications for that particular host or service. Contact groups can contain one or more individual contacts.
+
+When Alignak sends out a host or service notification, it will notify each contact that is a member of
+any contact groups specified in the ``contact_groups`` option of the service definition. Alignak is aware
+that a contact may be a member of more than one contact group, so it removes duplicate contact notifications before it does anything.
+
```

Comment:
Add some words because we can too have `contacts` defined directly in host / service for notifications